script/travis_elasticsearch_upgrade.sh

```diff
@@ -1,7 +1,9 @@
 #!/bin/bash
 cd /tmp
-wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.2.4.tar.gz
-tar xvfz /tmp/elasticsearch-6.2.4.tar.gz
```

Comment:
Can you move the version to a variable `ES_VERSION`, like the Codeship script?

Reply:
Indeed I can! (Fair warning, this is going to be a force push so this comment will probably disappear.)

Reply:
...false alarm 🎉